### DISCUSSION TITLE
About Radiologist Assessment

### DISCUSSION THREAD

**Discussion by mclvnot** [facebookresearch/fastMRI#316] [Q&A]
Hello,

For radiologist assessment, in sharpness, cnr and artefact,  how would radiologists evaluate dataset in terms of artefact? They vote hallucinations or not?

Best,

**Comment by mmuckley** [comment]
Hello @mclvnot, we supplied the radiologists with a rubric for their Likert score rating. They gave the rating that they thought best matched with the image.

We did not have a Likert rating for hallucinations, but we did allow radiologist comments. Typically, if the radiologist saw a hallucination, they would note it in their comments for that case.

This is for the 2020 fastMRI Challenge study.

**Comment by mclvnot** [comment]
I mean in Advancing machine learning for MR image reconstruction with an open competition: Overview of the 2019 fastMRI challenge, there is voting in Table 3 about artifact. What did radiologists do while evaluating artifacts? or which artifacts? (Aliasing etc?)

Best,

**Reply by mmuckley** [reply]
@mclvnot that table was essentially done the same way. They look at the images and evaluate it vs. a pre-specified rubric. An example rubric is documented in your previous question on #294.

**Reply by mclvnot** [reply]
Yes, I know your likert scale and for my study for citation these articles, I try to emphasize the differences. I think it is evaluated for aliasing artifact, according to reference, it was voted. Am I wrong?

Best,

**Reply by mmuckley** [reply]
@mclvnot In that paper "artifacts" would consider any type of artifact, not only hallucination. It would also consider blurring or motion as artifacts, for example.
